lib/typing/decimal/base.go

```diff
@@ -39,7 +39,7 @@ func (d Details) toKind(maxPrecision int32, exceededKind string) string {
 		return exceededKind
 	}
 
-	return fmt.Sprintf("NUMERIC(%v, %v)", d.precision, d.scale)
+	return fmt.Sprintf("NUMERIC(%d, %d)", d.precision, d.scale)
```

Comment:
Just calling out that this is technically a change.

Reply:
True, my bad - forgot to call this out